extension/src/test/suite/cli/child.ts

```diff
@@ -0,0 +1,25 @@
+import { getOptions } from './util'
```

Comment:
[F] The purpose of this script is to create a background process and give the pid back to the test.